src/Pyz/Zed/Merchant/Persistence/MerchantRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Pyz\Zed\Merchant\Persistence;
+
+use ArrayIterator;
+use Generated\Shared\Transfer\PyzMerchantEntityTransfer;
+use Orm\Zed\Merchant\Persistence\PyzMerchant;
+use Propel\Runtime\Collection\Collection;
+use Spryker\Zed\Kernel\Persistence\AbstractRepository;
+
+/**
+ * @method MerchantPersistenceFactory getFactory()
+ */
+class MerchantRepository extends AbstractRepository
+{
+    /**
+     * @param int[] $merchantIds
+     * @return ArrayIterator
+     */
+    public function queryMerchantsById(array $merchantIds): ArrayIterator
```

Comment:
`query`-prefix benutzen wir eigenlich nur in `queryContainer` wenn wir eine query ohne `find()` als return schicken.
hier währe `findMerchantsById ` passender.
Und ob `ById` oder `ByIdMerchant` da gibt es immer noch Diskussionen :)